src/BuildingRegistry.Producer.Snapshot.Oslo/ProducerBuildingProjections.cs

```diff
@@ -209,6 +209,18 @@ public ProducerBuildingProjections(IProducer producer, ISnapshotManager snapshot
                     ct);
             });
 
+            When<Be.Vlaanderen.Basisregisters.ProjectionHandling.SqlStreamStore.Envelope<BuildingUnitDeregulationWasCorrected>>(async (_, message, ct) =>
+            {
+                await FindAndProduce(async () =>
+                        await snapshotManager.FindMatchingSnapshot(
+                            message.Message.BuildingPersistentLocalId.ToString(),
+                            message.Message.Provenance.Timestamp,
+                            throwStaleWhenGone: false,
+                            ct),
+                    message.Position,
+                    ct);
+            });
+
```

Comment:
This shouldn't be here.
I can see others that shouldn't be here as well.

If you don't touch `BuildingV2` projections then this one shouldn't be touched either because it will never find a  matching snapshot.